_pages/onboarding.md

```diff
@@ -82,6 +82,8 @@ Welcome to 18F! We are so glad you are here.
 
 * Explore [employee discounts](../private/fed-discount-phone/), including about 15% on cellphone plans.
 
+* If you are not on the Delivery Engineering team, but are interested in attending monthly technical talks, email [Theresa Summa](mailto:theresa.summa@gsa.gov) and ask to be invited. You can also [submit a talk](https://docs.google.com/document/d/1FQLUofvibixHXg0iiUIdF9Up8djUrUL4xqU64sxHg-U/edit). If you are on the Delivery Engineering team, you will be invited automatically.
```

Comment:
How about "ask in #dev to be added to the calendar invite"? `channels > people` :wink: Otherwise :shipit: